src/appMain/hmi_capabilities.json

```diff
@@ -300,6 +300,12 @@
             },
             "phoneCapability": {
                 "dialNumberEnabled": true
+            },
+            "videoStreamingCapability": {
+                "supportedFormats": [{
```

Comment:
Sure, added in https://github.com/smartdevicelink/sdl_core/pull/1700/commits/9da6a4c5370008705ae39ad2e6fbbd73db9faaf3. If you have any sample values for these, please kindly suggest them.